src/main/java/fi/aalto/cs/apluscourses/intellij/actions/AboutAction.java

```diff
@@ -3,12 +3,17 @@
 import com.intellij.openapi.actionSystem.AnActionEvent;
 import com.intellij.openapi.project.DumbAwareAction;
 import fi.aalto.cs.apluscourses.ui.AboutDialog;
+import java.io.IOException;
 import org.jetbrains.annotations.NotNull;
 
 public class AboutAction extends DumbAwareAction {
 
   @Override
   public void actionPerformed(@NotNull AnActionEvent e) {
-    AboutDialog.display();
+    try {
+      AboutDialog.display();
+    } catch (IOException ioException) {
+      ioException.printStackTrace();
```

Comment:
I suggest considering if this is the correct level to handle the exception. In other words, perhaps it should be handled in the `display()` method already (or even deeper)?

Reply:
However, if you wish to still handle it here, why don't you try the:
```java
//top level declaration
private static final Logger logger = LoggerFactory.getLogger(AboutAction.class);

//instead of ioException.printStackTrace();
logger.error("An exception was thrown in an asynchronous call", ioException);
```

Reply:
Exception handling can probably be avoided altogether; see @nikke234's comment: https://github.com/Aalto-LeTech/intellij-plugin/pull/470#discussion_r556722429